docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+# Offloading multiple pods
```

Comment:
Offloading pods on multiple remote clusters at once

Reply:
Additional comment: I would update the image with the names of the clusters (e.g., destination cluster #1, etc.)

Reply:
Maybe just **Offloading on multiple remote clusters** would be better to avoid double lines in the documentation menù?

![image](https://user-images.githubusercontent.com/33266330/214828703-a6354598-42b6-47af-8f8a-c16853463ff4.png)

![image](https://user-images.githubusercontent.com/33266330/214828813-0c6addec-66ac-4157-9091-548fa7c98152.png)